src/alto-ui/Form/InputNumber/InputNumber.js

```diff
@@ -79,15 +81,17 @@ class InputNumber extends React.Component {
   render() {
     const { forwardedRef, locale, precision, disableThousandSeparator, ...rest } = this.props;
     return (
-      <TextField
-        ref={forwardedRef}
-        {...rest}
-        type="text"
-        value={this.state.display}
-        onChange={this.handleChange}
-        onFocus={this.handleFocus}
-        onBlur={this.handleBlur}
-      />
+      <div className="InputNumber">
```

Comment:
give the className directly to TextField and dont forget to merge it with incoming className prop